pkg/webui/console/views/application/index.js

```diff
@@ -151,6 +152,14 @@ export default class Application extends React.Component {
     startStream(appId)
   }
 
+  componentDidUpdate (prevProps) {
+    const { application, redirectToList } = this.props
+
+    if (Boolean(prevProps.application) && !Boolean(application)) {
+      redirectToList()
```

Comment:
The redirect results in an error for me, originating from the tabular component.
![image](https://user-images.githubusercontent.com/5710611/60253070-c0169800-98cb-11e9-9477-32c757c29d18.png)